test/tools/yulopti.cpp

```diff
@@ -331,20 +485,25 @@ int main(int argc, char** argv)
 			return 1;
 		}
 
+		if (arguments.count("object"))
+			objectPath = arguments["object"].as<string>();
```

Comment:
We should report an error when this is non-empty but the input is a block, not an object.

Reply:
I've added this check in `parse()`